Docker/README.md

```diff
@@ -49,6 +49,13 @@ Start 3 MPI nodes:
 ```bash
     source docker-compose-openmpi.sh up --scale mpi_head=1 --scale mpi_node=3
 ```
+
+**Testing OpenMPI with optional custom ssh_key for Borg** 
+
+Note that without specifying a location (the BITBUCKET_SSH_KEY), it will assume the ssh for bitbucket is set up in the standard location (e.g. `~/.ssh/id_rsa`)
+```bash
+    BITBUCKET_SSH_KEY=/path/to/id_rsa source docker-compose-openmpi.sh up --scale mpi_head=1 --scale mpi_node=3
+```
 Note this will call `Docker/build.sh`, so no need to do both.
```

Comment:
This line needs to go right below the command it is referencing.  Your new section should come after this line.  Otherwise it looks good.